common/conf/env/DEVINFRA.properties

```diff
@@ -57,6 +57,7 @@ commercial.travel_url=http://www.guardianholidayoffers.co.uk
 
 images.media.token=not-a-real-token
```

Comment:
> Aren't they there for the tests, otherwise they break?

The build agents have a `frontend.properties` file which overrides values in `DEVINFRA.properties`, so these values never actually got used.